pkg/status/templates/orchestrator.tmpl

```diff
@@ -31,8 +31,15 @@ Orchestrator Explorer
   ======================
   Orchestrator Endpoints
   ======================
-  {{- range $key, $value := .OrchestratorEndpoints}}
-  {{$key}} - API Key ending with: {{$value}}
+  {{- range $key, $values := .OrchestratorEndpoints}}
+  {{- if gt (len $values) 1}}
+  {{$key}} - API Keys ending with:
+  {{- range $values }}
+      - {{ . }}
+  {{- end}}
+  {{- else}}
+  {{$key}} - API Key ending with: {{index $values 0}}
```

Comment:
What is the purpose of `index` and `0`?

Reply:
values is an array of all api keys.
If the array contains more than one element we iterate over and add one line per api-key.
If the array contains a single element, we use `index $values 0` to retrieve the first item and display it.